### PR TITLE
Bootstrap3

### PR DESCRIPTION
**Pull request by stuartnelson3**
can you look at the date picker for the `until` time? not sure where it's getting called from, and it's appearing in the wrong location.

### CODE REVIEW

**Comment by stuartnelson3** [comment]
@juliusv

**Comment by juliusv** [comment]
Don't worry about the datepicker location for now - it was already off in the old version. But with Bootstrap 3, it doesn't look right anymore (the time selection area is just gray).

Looks like the datepicker is called from here: https://github.com/prometheus/prometheus/blob/master/web/static/js/graph.js (search for `appendDtpicker`)

**Comment by juliusv** [comment]
Actually the golden solution would be to replace this ugly datepicker with the one we have in PromDash :) But no worries if you want to keep that for a later change.

File: web/static/js/graph.js [hunk 1]
```diff
@@ -242,6 +244,13 @@ Prometheus.Graph.prototype.getEndDate = function() {
   if (!self.endDate || !self.endDate.val()) {
     return null;
   }
+  //
+  // This is returning the time set according to the system clock,
+  // but the datetimepicker is converting this to UTC, which is
+  // causing the time to decrease instead of increase. It should
+  // be increasing by 30min, but UTC is one hour behind berlin time,
+  // so incrementing UTC by 30min e.g. decrements the time the user sees by 30min.
+  //
```
Comment (stuartnelson3) [inline web/static/js/graph.js]: Can you look into this? My machine is returning a time that's an hour behind, even though the returned date object claims it is UTC + 1. * sigh * js dates.

**Comment by stuartnelson3** [comment]
@juliusv question about timezones, but other than that it appears to be working with the datetimepicker from promdash

**Comment by juliusv** [comment]
Hmm, seems another thing is broken with the timepicker? When I click the "Clock" symbol to set the time, not the data, it doesn't do anything and I get a JS error:

    Uncaught TypeError: undefined is not a function

**Comment by stuartnelson3** [comment]
Found the clock button error — just had to include `bootstrap.min.js`. We can look at the date tomorrow if you don't get to it today.

File: web/static/js/graph.js (hunk 1 above)
Comment (juliusv) [inline web/static/js/graph.js]: Changing it to this works:

    return self.endDate.data('datetimepicker').getDate().getTime();

Magic. :sparkles:

**Comment by juliusv** [comment]
@stuartnelson3 ok, try and see if you can still break anything now. The URL-updating/restoring behavior should now also work much better than before.

**Comment by stuartnelson3** [comment]
Typeahead is working in both os x firefox and chrome in the two situations (hard refresh and opening new tab)

**Comment by juliusv** [comment]
Cool, I pushed some more final cleanups and would be ok with merging it now if you are (despite not everything being super clean, because before, it wasn't either).

**Pull request merged**